usm_wrappers/salt_wrapper.py

```diff
@@ -63,6 +73,53 @@ def accept_minion(minion_id):
     return (True if out else False)
 
 
+def get_started_minions(minions=[], timeout=60):
+    minion_set = set(minions)
+    up_minions = set(runner.cmd('manage.up', []))
```

Comment:
timeout may not be honored if this takes a long time. This would be surprising to me.